test/functional/basic/touch-id-e2e-specs.js

```diff
@@ -32,29 +32,26 @@ if (!process.env.REAL_DEVICE) {
       await killAllSimulators();
     });
 
-    it('should throw an error if allowTouchIdEnroll desired capability is not set', async () => {
-      caps = Object.assign(TOUCHIDAPP_CAPS);
-      caps.allowTouchIdEnroll = false;
-      driver = await initSession(caps);
-
-      await driver.toggleTouchIdEnrollment().should.be.rejectedWith(/enroll touchId/);
-    });
-
     describe('touchID enrollment functional tests applied to TouchId sample app', function () {
-      beforeEach(async () => {
-        caps = Object.assign(TOUCHIDAPP_CAPS);
-        caps.allowTouchIdEnroll = true;
-        driver = await initSession(caps);
+      beforeEach(async function () {
+        if (process.env.TRAVIS) {
+          // ignore tests on Travis, since Appium process does not have access to
+          // system Accessibility there
+          this.skip();
```

Comment:
This needs a `return`, or the function continues. No need to create the session and all.